### PR TITLE
Deliverable 2

### PR DESCRIPTION
**Pull request by manfredovaldes**

### CODE REVIEW

File: src/components/ProductList.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            </li>)
+          )}
+        </ul>
+      </div>
+      <article className="App-product-list">
+        <h1>This is the Product List Page</h1>
+        <ul className="App-products">
+          {productsData.results.map(item => 
+            (<li key={item.id}>
+              {/* {JSON.stringify(item)} */}
```
Comment (julianzdelat) [inline src/components/ProductList.jsx:24]: delete this commented code

**Review by julianzdelat — CHANGES_REQUESTED** [review]
I don't see the filtering working for the categories on the preview URL, and I leave some comments here

File: src/components/ProductList.jsx (hunk 1 above)
Comment (manfredovaldes) [reply src/components/ProductList.jsx]: Done

File: src/App.js [hunk 2]
```diff
@@ -1,19 +1,30 @@
...
 import Header from './components/Header';
 import Footer from './components/Footer';
 import Home from './views/Home';
 
+import './App.css';
+import ProductList from './components/ProductList';
+
 function App() {
   const { data, isLoading } = useFeaturedBanners();
   console.log(data, isLoading);
```
[the remark this answers is not in the document]
Comment (manfredovaldes) [reply src/App.js]: Done

**Comment by manfredovaldes** [comment]
@julianzdelat Yeah, my bad.
I just uploaded these changes both to the repo and to the test site.

On a side note, apparently, all the products from the json catalogs are from the `furniture` category only.
So the rest of the categories are empty.

**Pull request merged**